WebHostLib/downloads.py

```diff
@@ -92,6 +92,8 @@ def download_slot_file(room_id, player_id: int):
             fname = f"AP_{app.jinja_env.filters['suuid'](room_id)}_P{slot_data.player_id}_{slot_data.player_name}.zip"
         elif slot_data.game == "Final Fantasy Mystic Quest":
             fname = f"AP+{app.jinja_env.filters['suuid'](room_id)}_P{slot_data.player_id}_{slot_data.player_name}.apmq"
+        elif slot_data.game == "The Wind Waker":
```

Comment:
webhost stuff should only be needed if  you don't use the AP Containers. Any particular reason why you don't? This list and the one in macros is meant to be reduced and eventually go away.

Reply:
Not really. I wasn't sure how to get the files to be downloadable on the web host. I will look into AP Containers.